01 OpenMW/Data Files/scripts/TamrielData/player.lua

```diff
@@ -0,0 +1,32 @@
+local types = require('openmw.types')
+local I = require('openmw.interfaces')
+local self = require("openmw.self")
+local ui = require("openmw.ui")
+
+local function tdSetEquipment(equipSlot)
+    -- Get current equipment
+    local equipped = types.Actor.equipment(self)
+
+    -- Check what slot do we want to unequip, then unequip it
+    if equipSlot == types.NPC.EQUIPMENT_SLOT.Boots then
+        equipped[types.Actor.EQUIPMENT_SLOT.Boots] = nil
```

Comment:
You could move these ` = nil` bits out of the if statements and unconditionally do `equipped[equipSlot] = nil`.

It might also be worthwhile to only show the message if the actor is a player.

Reply:
Re: the second point, I can add the check, but at the moment the script is only attached to players, not NPCs. We could also attach it to other actors, but I'm not sure if that's necessary when we can just make sure that no Imgas are wearing boots/helmets in the ESM.

Reply:
Oh I see. In that case it'd make sense to do `if not types.Player.objectIsInstance(actor) then return end` in the item handler.

I figured it'd apply to all actors to prevent the player from dressing up corpses, but I suppose that's not really worth doing.

Reply:
- Moved the checks out of the if statement, do `equipped[equipSlot] = nil` unconditionally
- Added the `if not types.Player.objectIsInstance(actor) then return end ` check to the item handler